src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -605,7 +605,7 @@ private function createResponsiveSizer(
         $style = 'padding-top:%s%%'
     ) {
         $padding       = $height->getNumeral() / $width->getNumeral() * 100;
-        $paddingString = (string) round($padding, 4);
```

Comment:
Why not do a `str_replace(',', '.', (string)round(...));`?

Reply:
Because the conversion to string uses the locale for the conversion. So, for example, the `,` could be used for separating the thousands.

Reply:
Why are the trailing zeros an issue in the first place? And what about applying a `trim($numberWithTrailingZeros, '0')`?

Reply:
This is part of an "optimizer" tool. So leaving around useless additional bytes in the markup is not great as part of the "optimization" process.

And indeed, doing 2 `rtrim()` operations seems to be more performant than the regex, as shown here: https://twitter.com/JanSlabon/status/1333402043915128833